### ISSUE TITLE
The loss value is about 5

### ISSUE THREAD

**Issue by mshmoon** [CoinCheung/BiSeNet#45]
I use this code to train camvid，and I get loss value is about 5, is this normal?

**Comment by CoinCheung** [comment]
I have never tried this on camvid, maybe you can spend some time tuning the hyper parameters.

**Comment by mshmoon** [comment]
You are very nice, old iron.
I have modified your code  a little, and set total epochs=700, the lossp is about 0.2.
I think bisenet(your) is better than ICNet.
But I have a doubt, I have studied semantic segmentation 2 years, many model be convergence only need about 30 epochs, but the bisenet need a lot of epochs.

**Comment by CoinCheung** [comment]
I have not met this, have you tried other learning rate or did you use proper lr_scheduler ?

**Comment by mshmoon** [comment]
Oh, no ,I use learning decay of every epoch

**Comment by CoinCheung** [comment]
I am closing this since there is no more question about the code.
